script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt

```diff
@@ -411,142 +473,161 @@ Arcueid stops walking.
-'This is bad. Don't take any more. You'll break next time. Your entire foundation will collapse. If that happens, you won't be able to recover.'
+'This is bad.
+ Don't take another hit.
+ You'll break next time.
+ The foundation of your self will collapse.
+ If that happens, you won't be able to recover.'
 }
 [sha:2093b0f5706b893c1c24d5cf1c3fc4268bcbba5e]{
 -- Page 308, Offset 39256.
 -- 「もう……黙っててよ、わかって、る、から……！」
-"Geez... Be quiet. I already... know... that...!"
+"Ugh... shut up already... I already... know... that!"
 }
 [sha:6d7e87bb85d701ab19f7748ab6ba900e76f6f4fc]{
 -- Page 308, Offset 39257.
 -- シエルは唇を噛んで弱音を呑み込み、次弾に備えた。
-Ciel bites down on her lips, swallowing down on her complaints before preparing for the next round.
+Ciel bites her lip,
+ silences her complaints,
+ and readies herself for the next strike.
 }
 [sha:b36c548a4fd818be0107f95303fb387d55d954f9]{
 -- Page 308, Offset 39258.
 -- 躱す事はできる。あんな大振りの攻撃になんて当たる筈がない。でもここからは動けない。
-She can avoid it. There's no way that large attack should be able to hit her. But she can't move from there.
+She could dodge it.
+ She'd have no problem evading a massive sweeping attack like that.
+ But she can't move from this spot.
 }
 [sha:590e3f68e7354d47453a36f199bdb7cb5e36a6de]{
 -- Page 308, Offset 39259.
 -- なぜなら、背後には<志|・><貴|・><の|・><待|・><つ|・><学|・><校|・><が|・><あ|・><る|・>。
-That's because Shiki is waiting in the school behind her.
+That's because the school――and Shiki――are right behind her.
 }
 [sha:04c5dabf2ba4e28a14530ed453e0781cf89a3d57]{
 -- Page 309, Offset 39260.
 -- あの衝撃波は街を削り、学校を一息で呑み込むものだ。
-That shockwave would wipe out the city and annihilate the school in one fell swoop.
+That shockwave could rip through the city and obliterate the school in the blink of an eye.
 }
 [sha:4695ac34419823b7b842e55016b9a830561003e4]{
 -- Page 309, Offset 39261.
 -- だから動けない。シエルはもう、ここであの星の息吹を防ぎ続ける事しか許されない……！
-That's why she can't move. She has no other option but to continue defending against that planet's breath...!
+That's why she can't move.
+ She has no option but to continue to serve as a bulwark against the Breath of the Planet!
 }
 [sha:43d9f65a5ff4b4a05a9284419f732ee2ba140236]{
 -- Page 310, Offset 39262.
 -- 「すごい―――すごいすごい、シエルすごい！
-"Incredible―――That's incredible, Ciel!
+"Incredible... Incredible! That was incredible, Ciel!
 }
 [sha:223239d84c2f845b09e03fa0ea62d498a1c8f945]{
 -- Page 310, Offset 39263.
 -- ねえどうなってるのシエル!?　これでも千切れないなんて、どこまで耐えられるのシエル！」
-Hey, what's going on Ciel?! To think that you're still standing after all that... just how long can you last?!"
+Hey, hey, how'd you do it?
+ I can't believe that wasn't enough to break you...
+ Just how much more can you take, Ciel?!"
 }
 [sha:94ebccb3f57fc65bcc712aea305e1af25ad3baf0]{
 -- Page 311, Offset 39264.
 -- 絶え間なく放たれる星の息吹。
-The next breath of the planet gets released without delay.
+The Breath of the Planet is unleashed again and again.
 }
 [sha:316b17d873f110a4a2811d949da996c2965e8ff8]{
 -- Page 311, Offset 39265.
```

Comment:
Sounds kinda off imo. Like she brute forced it (took multiple attempts)

Reply:
Right, the verb-form is only used for computational stuff.
Regardless, it is wrong right now, since it sounds like she's used up all her strength, when it's more that she just sort of put her arms into it and hoped for the best.

>Taking the first attack was a gamble achieved solely by brute force.